src/pages/ReportDetailsPage.tsx

```diff
@@ -9,7 +9,6 @@ import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView
 import Button from '@components/Button';
 import ConfirmModal from '@components/ConfirmModal';
 import DecisionModal from '@components/DecisionModal';
-import DelegateNoAccessModal from '@components/DelegateNoAccessModal';
```

Comment:
hmm the changes in this file are not behind the beta, are they? That means that users won't have a way to perform some of these actions....

Reply:
@luacmartins I think we're ready to merge now